src/main/java/com/internet/shop/dao/OrderDaoImpl.java

```diff
@@ -0,0 +1,45 @@
+package com.internet.shop.dao;
+
+import com.internet.shop.db.Storage;
+import com.internet.shop.lib.Dao;
+import com.internet.shop.models.Order;
+import java.util.List;
+import java.util.Optional;
+
+@Dao
+public class OrderDaoImpl implements OrderDao {
+
+    @Override
+    public Order create(Order order) {
+        return Storage.addOrder(order);
+    }
+
+    @Override
+    public Optional<Order> get(Long id) {
+        return Storage.orders.stream()
+                .filter(order -> id.equals(order.getId()))
+                .findFirst();
+    }
+
+    @Override
+    public List<Order> getAll() {
+        return Storage.orders;
+    }
+
+    @Override
+    public Order update(Order order) {
+        for (Order or : Storage.orders) {
+            if (or.getId().equals(order.getId())) {
+                Storage.orders.set(Storage.orders.indexOf(or), order);
+                return or;
+            }
+        }
+        throw new RuntimeException("Order does not exist in database");
```

Comment:
```suggestion
        throw new RuntimeException("Order does not exist in database with id " + order.getId);
```
повідомлення має бути інформативним

Reply:
Исправил во всех даошках